gateways/adyen/request_builder.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/adyen/adyen-go-api-library/v2/src/checkout"
 	"github.com/adyen/adyen-go-api-library/v2/src/payments"
 	"strconv"
+	"time"
+)
+
+const (
+	maxLineItemDescriptionLength = 26
+	timeFormatExample            = "020106"
```

Comment:
I believe this is a constant in the time package?

Reply:
Wasn't able to find a constant for this specific format but not using this anymore so not a concern